site/pages/home.html

```diff
@@ -52,6 +52,7 @@ <h2>Contributing</h2>
 			<ul>
 				<li><a href="{{site.github.repository_url}}/blob/master/CONTRIBUTING.md#contributing" class="o-typography-link--external">Read the contributing guide</a></li>
 				<li><a href="{{site.github.repository_url}}/issues/new" class="o-typography-link--external">Open an issue on GitHub</a></li>
+				<li><a href="{{site.github.repository_url}}/issues/new" class="o-typography-link--external">Read about the Engineering Progression Working Group</a></li>
```

Comment:
this is the wrong URL :) and it probably shouldn't have `o-typography-link--external`

Reply:
derpaderp